api/v2.0/swagger.yaml

```diff
@@ -1176,11 +1176,11 @@ paths:
         - $ref: '#/parameters/projectName'
         - $ref: '#/parameters/repositoryName'
         - $ref: '#/parameters/reference'
-        - name: scan_request_type
+        - name: scan_type
```

Comment:
Better to rename the param name from 'scan_type' to 'scanType'
To keep it with the same naming convention as :
        - $ref: '#/parameters/projectName'
        - $ref: '#/parameters/repositoryName'
        - $ref: '#/parameters/reference'

Reply:
updated